Bonsai.Harp.CF/SyringePumpCommand.cs

```diff
@@ -122,49 +120,47 @@ protected override Expression BuildSelector(Expression expression)
             }
         }
 
-        static HarpMessage ProcessEnableMotorDriver() => HarpCommand.WriteByte(address: 32, 1);
-        static HarpMessage ProcessDisableMotorDriver() => HarpCommand.WriteByte(address: 32, 0);
-        static HarpMessage ProcessStartProtocol() => HarpCommand.WriteByte(address: 33, 1);
-        static HarpMessage ProcessStopProtocol() => HarpCommand.WriteByte(address: 33, 0);
+        static HarpMessage ProcessSetMotorDriver(bool input) => HarpCommand.WriteByte(32, (byte) (input ? 1 : 0));
+        static HarpMessage ProcessSetDirection(bool input) => HarpCommand.WriteByte(35, (byte) (input ? 1 : 0));
+        static HarpMessage ProcessEnableProtocol(bool input) => HarpCommand.WriteByte(33, (byte) (input ? 1 : 0));
         static HarpMessage ProcessSetDigitalOutputs(byte input) => HarpCommand.WriteByte(address: 39, input);
         static HarpMessage ProcessClearDigitalOutputs(byte input) => HarpCommand.WriteByte(address: 40, input);
         static HarpMessage ProcessMotorMicrostep(MotorMicrostep input)
         {
-            bool exists = Enum.IsDefined(typeof(MotorMicrostep), input);
-            if (!exists)
-                throw new InvalidOperationException("Invalid Mask selection. Please select an appropriate value.");
+            if(!Enum.IsDefined(typeof(MotorMicrostep), input))
+                throw new InvalidOperationException("Invalid MotorMicrostep value. Valid values are: \nFull = 0\nHalf = 1\nQuarter = 2\nEighth = 3\nSixteenth = 4.");
 
-            return HarpCommand.WriteByte(address: 44, (byte)input);
+            return HarpCommand.WriteByte(address: 44, (byte) input);
         }
 
         static HarpMessage ProcessProtocolNumberOfSteps(ushort input)
         {
             if (input <= 0)
-                throw new InvalidOperationException("Invalid number of steps. Must be above 0.");
+                throw new InvalidOperationException("Invalid number of steps. Must be greater than 0.");
 
             return HarpCommand.WriteUInt16(address: 45, input);
         }
 
         static HarpMessage ProcessProtocolStepsPeriod(ushort input)
         {
-            if (input <= 0)
-                throw new InvalidOperationException("Invalid steps period. Must be above 0.");
+            if (input < 1)
+                throw new InvalidOperationException("Invalid steps period. Must be greater or equal than 1.");
```

Comment:
Maybe change this to the same error as above (line 139) for consistency? I find it more readable to say "must be greater than zero" than "must be greater or equal than one".